migrations/2018_01_15_072600_change_email_tokens_b8_columns.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Schema\Builder;
+
+return [
+    'up' => function (Builder $schema) {
+        $schema->table('email_tokens', function (Blueprint $table) {
+            $table->renameColumn('id', 'token');
+        });
+    },
+
+    'down' => function (Builder $schema) {
+        $schema->table('email_tokens', function (Blueprint $table) {
+            $table->renameColumn('token', 'id');
+        });
+    }
+];
```

Comment:
This could use the `Migration::renameColumn` helper? (I'm starting to think these helpers aren't that valuable, since they can only be used in such limited instances)